src/tests/ftest/util/ServerUtils.py

```diff
@@ -33,18 +33,126 @@
 import signal
 import fcntl
 import errno
+import yaml
 
 from avocado.utils import genio
 
 sessions = {}
 
+DEFAULT_YAML_FILE = "utils/config/examples/daos_server_sockets.yml"
+AVOCADO_YAML_FILE = "utils/config/daos_avocado_test.yml"
+
 class ServerFailed(Exception):
     """ Server didn't start/stop properly. """
 
 # a callback function used when there is cmd line I/O, not intended
 # to be used outside of this file
 def printFunc(thestring):
-        print("<SERVER>" + thestring)
+    print("<SERVER>" + thestring)
+
+def nvme_yaml_config(default_yaml_value, bdev, enabled=False):
+    """
+    Enable/Disable NVMe Mode.
+    By default it's Enabled in yaml file so disable to run with ofi+sockets.
+    """
+    if 'bdev_class' in default_yaml_value['servers'][0]:
+        if default_yaml_value['servers'][0]['bdev_class'] == bdev and not enabled:
+            del default_yaml_value['servers'][0]['bdev_class']
+    if enabled:
+        default_yaml_value['servers'][0]['bdev_class'] = bdev
+
+def remove_mux_from_yaml(avocado_yaml_file, avocado_yaml_file_tmp):
+    """
+    Function to remove "!mux" from yaml file and create new tmp file
+    Args:
+        avocado_yaml_file: Avocado test yaml file
+        avocado_yaml_file_tmp: Avocado temporary test yaml file
+    """
+    with open(avocado_yaml_file, 'r') as rfile:
+        filedata = rfile.read()
+    filedata = filedata.replace('!mux', '')
+    with open(avocado_yaml_file_tmp, 'w') as wfile:
+        wfile.write(filedata)
+
+def update_server_options(avocado_yaml_value, default_yaml_value):
```

Comment:
I wonder if we should just have a table built into the code that has default values we want for testing rather than reading from example file.   That would create a maintenance issue where we have to keep our default values correct, but not clear to me they will keep the example file up to date either.  Then anything in the yaml file over lays our defaults.

Reply:
May be we can copy the example file to src/test/ftest and use that as reference so we do not have to worry if example  file get changed. Thoughts?